packages/smooth_app/pubspec.yaml

```diff
@@ -77,8 +82,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   mockito: ^5.1.0
-  path_provider_platform_interface: ^2.0.4
-  plugin_platform_interface: ^2.1.2
+  path: ^1.8.0 # careful with 1.8.1 because of https://github.com/flutter/flutter/issues/95478
```

Comment:
```suggestion
  path_provider_platform_interface: ^2.0.4
  plugin_platform_interface: ^2.1.2
  path: ^1.8.0 # careful with 1.8.1 because of https://github.com/flutter/flutter/issues/95478
```